{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2021-28950"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "An issue was discovered in fs/fuse/fuse_i.h in the Linux kernel before 5.11.8. A \"stall on CPU\" can occur because a retry loop continually finds the same bad inode, aka CID-775c5033a0d1."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.11.8",
          "refsource": "MISC",
          "tags": [],
          "url": "https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.11.8"
        },
        {
          "name": "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=775c5033a0d164622d9d10dd0f0a5531639ed3ed",
          "refsource": "MISC",
          "tags": [],
          "url": "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=775c5033a0d164622d9d10dd0f0a5531639ed3ed"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2021-03-22T11:54Z",
  "publishedDate": "2021-03-20T20:15Z"
}